charts/karpenter/templates/clusterrole-core.yaml

```diff
@@ -29,6 +29,9 @@ metadata:
   {{- end }}
 rules:
   # Read
+  - apiGroups: ["coordination.k8s.io"]
+    resources: ["leases"]
+    verbs: ["list", "watch"]
```

Comment:
I think you need "get" too, right?

Reply:
I was able to run the it, and all the e2etests passed without the `get`

Reply:
🤔 Might as well just add it. It's effectively the same. I think you are "technically" right that you don't need it since the "Get()" from controller-runtime is actually based on the in-memory cache that relies on the List/Watch. Still, I don't see a reason to differentiate this from our other read permissions.